packages/sourcecred/src/plugins/discord/config.test.js

```diff
@@ -6,7 +6,7 @@ describe("plugins/discord/config", () => {
   it("can load a basic config", () => {
     const raw = [
       {
-        guildId: "453243919774253079",
+        guildId: "453243919774253079//sourcecred",
```

Comment:
Could you add a test for when there's no delimiter since this should be optional and backwards compatible and since i fear the future updates to this